app/src/main/java/com/lightricks/feedexercise/data/FeedRepository.kt

```diff
@@ -20,18 +20,18 @@ class FeedRepository(private val feedApiService: FeedApiService) {
 
     fun fetchFeed(): Completable {
         return feedApiService.getFeed()
-                .subscribeOn(Schedulers.io())
-                .map { response -> response.templatesMetadata.map { it.toFeedItem() } }
-                .observeOn(AndroidSchedulers.mainThread())
-                .doOnSuccess { items -> feedItems.onNext(items) }
-                .ignoreElement()
+            .subscribeOn(Schedulers.io())
+            .map { response -> response.templatesMetadata.map { it.toFeedItem() } }
+            .observeOn(AndroidSchedulers.mainThread())
+            .doOnSuccess { items -> feedItems.onNext(items) }
+            .ignoreElement()
     }
 
     private fun TemplateMetadataItem.toFeedItem(): FeedItem {
         return FeedItem(
-                id = id,
-                thumbnailUrl = FeedApiService.THUMBNAIL_PREFIX + templateThumbnailURI,
-                isPremium = isPremium
+            id = id,
+            thumbnailUrl = FeedApiService.THUMBNAIL_PREFIX + templateThumbnailURI,
+            isPremium = isPremium
```

Comment:
I general as a reviewer I would not expect such change in such a PR. I we want to refactor a file we can do this but in a separated pr. Even more there if this file a no reason to change in this commit.